simplipy/camera.py

```diff
@@ -12,12 +12,14 @@
 DEFAULT_AUDIO_ENCODING = "AAC"
 
 CAMERA_MODEL_CAMERA = "camera"
+CAMERA_MODEL_CAMERA_1080 = "camera"
```

Comment:
We shouldn't have constants with the same value. Is there any reason we can't use the existing `CAMERA_MODEL_CAMERA`?

Reply:
No, that's a fair point. I'll remove that.